### ISSUE TITLE
Research and document text-based battle systems in existing games

### ISSUE THREAD

**Issue by adinilfeld** [uchicago-cs/chiventure#848]
Although there is currently a wiki page with [user stories](https://github.com/uchicago-cs/chiventure/wiki/RPG-Battle-Systems-%7C-User-Stories) for RPG battles, it would be helpful to have a wiki page with concrete examples of implementations of battles in other text-based RPGs, as well as a summary of which of their features we do or don't currently support.

**Comment by adinilfeld** [comment]
Started my research during Sprint 1, will type up my findings into a Wiki page during the beginning of Sprint 2.

**Comment by adinilfeld** [comment]
Summary of Spring 1 progress: 
- Found 5 examples of RPGs which feature mostly text-based combat: Wasteland, Dwarf Fortress, Bard's Tale, Wizardry, and Pokémon (though almost all of those have visuals supplementing the text)
- Compiled videos of combat in each game, for future reference
- Began listing notable characteristics of each combat system

I'm currently moving what I've found to a Google Doc, which I'll post here tomorrow. Then I just have to finish the lists of characteristics, mark which features we currently support, and finally put the info into a wiki page.

**Comment by adinilfeld** [comment]
Have moved my research so far to a Google Doc, available [here](https://docs.google.com/document/d/19vOvEbjkjn2gJwjT7XrCsGK5rYviKKZZDVESu3IM_oo/edit?usp=sharing) (commenting privileges only). Includes links to videos of battles in each RPG, notes about their combat systems, and screenshots. Tomorrow I'll move the info to a Wiki page, and then we should be good.

**Comment by adinilfeld** [comment]
Added all of the info to a wiki page: [RPG Battle Systems | Text Based Combat in Other Games
](https://github.com/uchicago-cs/chiventure/wiki/RPG-Battle-Systems-%7C-Text-Based-Combat-Systems-in-Other-Games) Hopefully it comes in handy!

Note: I decided not to mark which features which we do/don't currently support; our combat system is currently really barebones, so pretty much every line would be marked "not supported," which seems redundant. As we expand the battle system, it may be worth adding in notes about which features we support.
